File: 1181.py
linha = int(input())
tipo = input()
matriz = []
coluna = []
soma = 0
for i in range(0, 12):
    for a in range(0, 12):
        coluna += [float(input())]
    matriz += [coluna[:]]
    coluna.clear()
for i in range(0, 12):
    soma += matriz[linha][i]
if tipo == 'S':
    print(f'{soma:.1f}')
else:
    print(f'{soma/12:.1f}')

File: 1181.cpp
#include <iostream>
#include <iomanip>
using namespace std;
int main(){
    int linha;
    cin >> linha;
    string tipo;
    cin >> tipo;
    int matriz[12][12];
    double p;
    double soma = 0;
    for(int i = 0; i < 12; i++){
        for(int a = 0; a < 12; a++){
            cin >> p;
            matriz[i][a] = p;
        }
    }
    for(int i = 0; i < 12; i++){
        soma += matriz[linha][i];
    }
    if(tipo == "S"){
        cout << fixed << setprecision(1);
        cout << soma << endl;
    }
    else{
        cout << fixed << setprecision(1);
        cout << (soma/12) << endl;
    }
}
